### PR TITLE
Changed webidl generation to use data-lt attribute

### PR DESCRIPTION
**Pull request by halindrome**
This should address issue #464, but I don't have any specs with
webidl in them to verify that!  Can someone please take a close look?

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@jyasskin can you please take a look?

File: js/core/webidl-contiguous.js [hunk 1]
```diff
@@ -720,7 +720,7 @@ define(
                     // Found it: update the definition to specify its [for] and title.
```
Comment (jyasskin) [inline js/core/webidl-contiguous.js]: Update the comments too.

**Comment by jyasskin** [comment]
There's also a use of `@data-title` in `run()`. Can you double-check whether that needs to change?

Other than that and the comments, this looks good to me. Thanks.

**Comment by inexorabletash** [comment]
@marcoscaceres - any chance for progress here?

**Comment by marcoscaceres** [comment]
Merging this, if there are issues with `@data-title`, can fix them in a new bug. We should find out pretty quickly.

**Comment by halindrome** [comment]
Fwiw I looked at this and data-title is being used for something else.
Shouldn't be a problem.

**Comment by marcoscaceres** [comment]
Thanks for checking that @halindrome

**Pull request merged**